libraries/shared/factories/storage/transformer.go

```diff
@@ -39,7 +39,7 @@ func (transformer Transformer) KeccakContractAddress() common.Hash {
 	return transformer.HashedAddress
 }
 
-func (transformer Transformer) Execute(diff utils.StorageDiff) error {
+func (transformer Transformer) Execute(diff utils.PersistedStorageDiff) error {
```

Comment:
Minor, but I think `utils` is a weird package for absolutely core functionality-providing types :sweat_smile:

Reply:
`/storage/diff/` :question:

Reply:
I agree, but am reluctant to include that change in this PR since it cascades to affect the namespace for the `keys_lookup`, `keys_loader`, etc (and this PR is already going to require a lot of changes in the transformers) - thinking maybe we keep it as is in this PR and then have a separate one for cleaning up the package naming?